monkey/agent_plugins/credentials_collectors/chrome/src/chrome_credentials_collector.py

```diff
@@ -0,0 +1,63 @@
+import time
+from typing import Sequence
+
+from common.agent_events import CredentialsStolenEvent
+from common.credentials import Credentials
+from common.event_queue import IAgentEventPublisher
+from common.tags import (
+    CREDENTIALS_FROM_PASSWORD_STORES_T1555_TAG,
+    DATA_FROM_LOCAL_SYSTEM_T1005_TAG,
+    UNSECURED_CREDENTIALS_T1552_TAG,
+)
+from common.types import AgentID, Event
+
+from .typedef import CredentialsDatabaseProcessorCallable, CredentialsDatabaseSelectorCallable
+
+CHROME_CREDETIALS_COLLECTOR_TAG = "chrome-credentials-collector"
+CHROME_COLLECTOR_EVENT_TAGS = frozenset(
+    (
+        CHROME_CREDETIALS_COLLECTOR_TAG,
+        DATA_FROM_LOCAL_SYSTEM_T1005_TAG,
+        UNSECURED_CREDENTIALS_T1552_TAG,
+        CREDENTIALS_FROM_PASSWORD_STORES_T1555_TAG,
+    )
+)
+
+
+class ChromeCredentialsCollector:
```

Comment:
No tests for `ChromeCredentialsCollector`?

Reply:
+1